internal/node-patches/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
the LICENSE should get copied in from the root, don't want to have extra copies in each subdir

Reply:
yeah its not really needed now that we moved it into internal too.

Reply:
could you remove it before merging then?